### PR TITLE
/VL-5_Fix-icon-names_Dmytro-Holdobin

### PR DESCRIPTION
**Pull request by Explicit12**
https://ilevel.atlassian.net/jira/software/c/projects/VL/boards/11?selectedIssue=VL-5

### CODE REVIEW

File: src/ui.data-list/composables/attrs.composable.js [hunk 1]
```diff
@@ -14,9 +14,9 @@ export function useAttrs(props) {
   const itemWrapperAttrs = getAttrs("itemWrapper");
   const itemAttrs = getAttrs("item");
   const customActionsAttrs = getAttrs("customActions");
-  const iconDeleteAttrs = getAttrs("iconDelete", { isComponent: true });
-  const iconEditAttrs = getAttrs("iconEdit", { isComponent: true });
-  const iconDragAttrs = getAttrs("iconDrag", { isComponent: true });
+  const iconDeleteAttrs = getAttrs("deleteIcon", { isComponent: true });
```
Comment (itsJohnnyGrid) [inline src/ui.data-list/composables/attrs.composable.js:17]: iconDeleteAttrs - константи також треба перейменувати, щоб все було консистентно

File: src/ui.dropdown-list/index.vue [hunk 2]
```diff
@@ -82,7 +82,7 @@
             internal
             color="white"
             size="xs"
-            :name="config.iconAddName"
+            :name="config.addIconName"
             v-bind="iconAddAttrs"
```
Comment (itsJohnnyGrid) [inline src/ui.dropdown-list/index.vue:86]: here is also

File: src/ui.form-input-file/composables/attrs.composable.js [hunk 3]
```diff
@@ -56,7 +56,6 @@ export function useAttrs(props) {
   const placeholderAttrs = getAttrs("placeholder", { classes: placeholderClasses });
   const clearIconAttrs = getAttrs("clearIcon", { isComponent: true });
   const chooseFileIconAttrs = getAttrs("chooseFileIcon", { isComponent: true });
-  const chooseFileIconNameAttrs = getAttrs("chooseFileIconName", { isComponent: true });
```
Comment (itsJohnnyGrid) [inline src/ui.form-input-file/composables/attrs.composable.js:59]: why this one has been removed?

Comment (Explicit12) [reply src/ui.form-input-file/composables/attrs.composable.js:59]: Not used. It's icon name

Comment (itsJohnnyGrid) [reply src/ui.form-input-file/composables/attrs.composable.js:59]: ou, I see :)

**Review by itsJohnnyGrid — APPROVED** [review]

**Pull request merged**